docs/configuration.md

```diff
@@ -193,6 +193,25 @@ of the most common options to set are:
     and <code>spark.driver.memory</code>.
   </td>
 </tr>
+<tr>
+ <td><code>spark.resources.coordinate.enable</code></td>
+  <td>true</td>
+  <td>
+    Whether to coordinate resources automatically among workers/drivers(client only) 
+    in Standalone. If not, user should be responsible for assigning different resources 
```

Comment:
use same text as above config

If false, the user is responsible for configuring different resources for workers/drivers that run on the same host.